utils.py

```diff
@@ -0,0 +1,29 @@
+"""Utilities."""
```

Comment:
We probably don't need that in the utility package.
I think we should create some FIFO_VERBOSE global variable and unless it is true we should not print/log anything at all (except maybe in tests)
if FIFO_VERBOSE is true, then we should use just standard logger, not colorlog.

Bottom line: we don't want to introduce any unnecessary dependencies for the users of this package

Reply:
Have removed utils and shifted everything to the test file